picard/ui/colors.py

```diff
@@ -97,6 +104,22 @@ def register_color(themes, name, value):
 register_color(_LIGHT, 'first_cover_hl', 'darkgoldenrod')
 register_color(_DARK, 'first_cover_hl', 'orange')
 
+# syntax highlighting colors
+register_color(_LIGHT, 'syntax_hl_error', 'blue')
```

Comment:
Should we give the error a separate color? Maybe red.

Reply:
Yes, but I prefer to do tuning after we merged this. I intentionnally didn't change colors in this PR.

Reply:
I don't think we need different colors. The cases where the colors are currently the same makes. It's great that the colors can be configured separately, but I don't see why we need to have different colors by default.